<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compressionDialog</class>
 <widget class="QMainWindow" name="compressionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Compression</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainIcon.qrc">
    <normaloff>:/resources/icon/compress.ico</normaloff>:/resources/icon/compress.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="filePreview">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>File Preview</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPlainTextEdit" name="enTextArea">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Type or paste the text here.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="deTextArea">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Type or paste the text here.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="enCode">
            <property name="text">
             <string>Encode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveHCFile">
            <property name="text">
             <string>Save HuffmanCodings File (.hcf)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="deCode">
            <property name="text">
             <string>Decode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="openHCFile">
            <property name="text">
             <string>Open HuffmanCodings File (.hcf)</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="mainIcon.qrc"/>
 </resources>
 <connections/>
</ui>
